appengine-java8/taskqueues-push/src/main/java/com/example/appengine/taskqueue/push/Enqueue.java

```diff
@@ -43,9 +44,28 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 
     // Add the task to the default queue.
     // [START addQueue]
-    Queue queue = QueueFactory.getDefaultQueue();
-    queue.add(TaskOptions.Builder.withUrl("/worker").param("key", key));
-    // [END addQueue]
+    try (CloudTasksClient client = CloudTasksClient.create()) {
+      String projectId = "starter-akitsch";
```

Comment:
😲

Reply:
?! I have no idea how that file got added to this PR?!